docs/ref/templates/builtins.txt

```diff
@@ -903,24 +903,43 @@ attribute and calling the result ``country_list``.
 * ``list`` -- a list of all items in this group (e.g., a list of all cities
   with country='India').
 
+.. versionchanged:: 1.10
```

Comment:
Yes, that was a bad copy/paste on my part (I can never remember the `versionchanged` syntax :) ). Thanks